src/tf/TFClient.js

```diff
@@ -45,15 +64,16 @@ function TFClient(options) {
  *
  * @param tf - the TF message from the server
  */
-TFClient.prototype.processFeedback = function(tf) {
+TFClient.prototype.processTFArray = function(tf) {
   var that = this;
   tf.transforms.forEach(function(transform) {
     var frameID = transform.child_frame_id;
-    if (frameID[0] === '/') {
+    if (frameID[0] === '/')
+    {
       frameID = frameID.substring(1);
```

Comment:
Hmm, would it be worthwhile to just import a node module for it?

https://www.npmjs.com/package/underscore.string
https://www.npmjs.com/package/lodash.trimleft